src/tools/interop/interop.cpp

```diff
@@ -403,11 +409,15 @@ class InteropConnection {
         Streams.clear();
         Shutdown();
         MsQuic->ConnectionClose(Connection);
+        QuicEventUninitialize(TicketReceived);
         QuicEventUninitialize(ShutdownComplete);
         QuicEventUninitialize(RequestComplete);
         QuicEventUninitialize(QuackAckReceived);
         QuicEventUninitialize(ConnectionComplete);
         delete [] NegotiatedAlpn;
+        if (ResumptionTicket) {
+            delete [] ResumptionTicket;
```

Comment:
delete allows a nullptr to be passed in. No need to check before you call it.

Reply:
You didn't delete this if check.

Reply:
Missed one!